api/admin/controller/__init__.py

```diff
@@ -1564,9 +1564,12 @@ def _get_prior_test_results(self, item, protocol_class=None, *extra_args):
                     self._db, None, self._db, item
                 )
             elif self.type == "metadata service" and protocol_class:
-                self_test_results = protocol_class.prior_test_results(
-                    self._db, *extra_args
-                )
+                try:
+                    self_test_results = protocol_class.prior_test_results(
```

Comment:
This caused an error for the NovelistAPI (and a different class which I am forgetting right now), but the error was that it did not have the `prior_test_results` function.